submarine-workbench/workbench-web-ng/src/app/pages/workbench/notebook/notebook.component.ts

```diff
@@ -269,10 +255,12 @@ export class NotebookComponent implements OnInit {
       }
     }
 
+    console.log(newNotebookSpec);
```

Comment:
You can comment out this output log and open it when debugging?

Reply:
@xunliu  Thanks your suggestion. I have comment out that output log.